plugins/importexport/native/filter/NativeXmlSubmissionFileFilter.inc.php

```diff
@@ -127,7 +127,7 @@ function handleElement($node) {
 		if ($credit = $node->getAttribute('credit')) {
 			$submissionFile->setData('credit', $credit);
 		}
-		if ($directSalesPrice = $node->getAttribute('direct_sales_price')) {
+		if (strlen($directSalesPrice = $node->getAttribute('direct_sales_price'))) {
```

Comment:
in `main` branch the fix for that is 

```
 $directSalesPrice = $node->getAttribute('direct_sales_price');
        if ($directSalesPrice != null) {
            $submissionFile->setData('directSalesPrice', $directSalesPrice);
        }
```
I suppose here you want to have the same affect?

Reply:
I can't remember very well what was happening, I just remember there was an issue with `direct_sales_price="0"`.

As the `getAttribute()` always returns a string, I think the `strlen` has a clearer intention, so I think I'll leave it here and perhaps update the /main (if there's something here that needs to be ported there) 🤔